### ISSUE TITLE
SLZB-06 POE with CC2652P

### ISSUE THREAD

**Issue by Mrsash2020** [Andrik45719/Z-Stack-firmware#4]
Ok So I updated the firmware(Z-Stack-firmware 460800bps mod) but I must be doing something wrong. 

Read me says
SLZB-06
CC1352P2_CC2652P_other_coordinator_*.zip
After CC2652P firmware update set baudrate to 460800bps

System and Tools -> System Tools -> File Brouser -> File : configSerial.json
{"baud":460800,"port":6638}
Save

I did the above in the Zigbee2MQTT Edge addon config screen and it just would not work. If this configSerial.json is a file then where is it? 

![image](https://github.com/Andrik45719/Z-Stack-firmware/assets/62192773/a9766422-10cd-4e05-8ad2-e01efe31a189)

**Comment by Andrik45719** [comment]
You have to setup baud rate at SLZB-06 web interface

**Comment by Mrsash2020** [comment]
Yes it does it automatically in the Z2M and ZHA config generator for me to copy and also sets it in serial options
I also set the config for the addon as I said above. Basically the entire thing has gone to hell. lol Switches are not responding etc. Went back to previous config, but still stuffed.

Update: Ok so I successfully went to the Basic version

So I see there are a few places the baud rate is specified and not sure which/where else it needs to be amended etc. For eg
Its in the Z2M addon config screen in home assistant.
![image](https://github.com/Andrik45719/Z-Stack-firmware/assets/62192773/c2345918-8c47-4bfa-9726-be301e46eb6d)

Its /homeassistant/zigbee2mqtt/configuration.yaml file
![image](https://github.com/Andrik45719/Z-Stack-firmware/assets/62192773/4340a504-3319-4c7f-8e90-8d7e7753bc54)

http://slzb-06.local/z2m_zha link on the SLZB-06 screen
![image](https://github.com/Andrik45719/Z-Stack-firmware/assets/62192773/8609f390-cae1-4c7e-9c13-01b045395859)

Z2M web interface /settings/Serial
![image](https://github.com/Andrik45719/Z-Stack-firmware/assets/62192773/33bbdfb5-ca3d-4c0c-a577-22aa377f2b3d)

**Comment by raphael1688** [comment]
Maybe it's off the topic. Will you release this high baud rate version for 20240710? Thanks!

**Comment by mixedbreed** [comment]
> Maybe it's off the topic. Will you release this high baud rate version for 20240710? Thanks!

Support this!
I have a setup relying on very chatty presence sensors - the increased baudrate has been very helpfull.
Thanks!
